# Desired states from Joystick(Probably...)

# Desired wbr velocity : v_d
# Desired wbr yaw rate :yaw_rate_d
# Desired wbr height : h_d
# Desired wbr roll angle : phi_d

float64 v_d
float64 yaw_rate_d
float64 h_d
float64 phi_d